src/lib/operators/join_hash.cpp

```diff
@@ -371,6 +371,12 @@ class JoinHash::JoinHashImpl : public AbstractReadOnlyOperatorImpl {
       _performance.set_step_runtime(OperatorSteps::BuildSideMaterializing, timer_materialization.lap());
     }
 
+    // Store the number of probe values after the probe side has been materialized. Depending on the input sizes, the
+    // probe side has potentially been filtered using the Bloom filter.
```

Comment:
bloom vs Bloom (I'd use the former)

Reply:
Since its a name it should actually be capitalized. I'll change it to be "Bloom" everywhere, objection?